cvmfs/sync_union_tarball.h

```diff
@@ -66,19 +66,28 @@ class SyncUnionTarball : public SyncUnion {
   std::set<std::string>
       know_directories_;  ///< directory that we know already exist
 
-  ///< directories where we found catalog marker, after the main traverse we
-  ///< iterate through them and we add the catalog
+  /** 
+   * directories where we found catalog marker, after the main traverse we
+   * iterate through them and we add the catalog
+   **/
```

Comment:
The Doxygen syntax uses a single star at the end, I think.  Like

    /**
     * Comment
     */